src/db/dal/savedFilter.ts

```diff
@@ -4,12 +4,11 @@ import { Op } from 'sequelize';
 
 import SavedFilterModel, { ISavedFilterInput, ISavedFilterOutput } from '../models/SavedFilter';
 
-const sanitizeInputPayload = (payload: ISavedFilterInput) => ({
-    ...payload,
-    id: undefined,
-    keycloak_id: undefined,
-    creation_date: undefined,
-});
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const sanitizeInputPayload = (payload: ISavedFilterInput) => {
+    const { id, keycloak_id, creation_date, ...rest } = payload;
```

Comment:
I think we should explicitly extract the needed fields (implicit vs explicit)

Reply:
Non ici le but est de omit ces fields, pas de relister tout le typage du model ISavedFilterInput.
Si on rajoute des fields dans ISavedFilterInput plus tard pour divers raison il faudra alors modifié partout où dans le code on a extrait explicitement les fields; c'est pas pratique.
C'est pourquoi on utilise ...rest (plus souvent appelé otherFields) avec cette facon implicit comme tu dis.

Btw ici c'est un merge de ce qui avait dans users-api de include, on a pas refacto et réécrit tout le code, il vient pas de nous. Mais on fait les corrections de code qu'on juge necessaire ou non risqué bien sur